features/accommodations/creation/AccommodationMultiStepForm.tsx

```diff
@@ -123,6 +126,8 @@ export const AccomodationMultiStepForm: FC = () => {
   function onSubmit(e: FormEvent) {
     e.preventDefault();
     if (!isLastStep) return next();
+
+    data.priceDiffs.forEach((pd) => (pd.id = pd.id?.includes(".") ? undefined : pd.id));
```

Comment:
Each added price diff has a random number for id so that the React node can have a key prop other than the index.
priceDiffs: [  ...priceDiffs, { percentage: 0, when: "0 0 * * *", id: Math.random() + "" } ]
Here we are removing that temporary id.

Reply:
But since this form is only for creation, the id will be ignored anyway, so we can remove this line.